packages/theme/src/cli/utilities/theme-environment/theme-polling.test.ts

```diff
@@ -179,6 +173,7 @@ describe('pollRemoteJsonChanges', async () => {
       await pollRemoteJsonChanges(developmentTheme, adminSession, remoteChecksums, defaultThemeFileSystem)
 
       // Then
+      expect(fetchThemeAsset).not.toHaveBeenCalled()
```

Comment:
this test was not failing and I realized it's because `fetchThemeAsset` is returning `undefined`